maldoca/base/file.h

```diff
@@ -29,6 +30,7 @@
 #include "absl/strings/str_join.h"
 #include "absl/strings/string_view.h"
 #include "google/protobuf/message.h"
+#include "maldoca/base/get_runfiles_dir.h"
```

Comment:
./maldoca/base/file.h:33:10: fatal error: 'maldoca/base/get_runfiles_dir.h' file not found
#include "maldoca/base/get_runfiles_dir.h"

https://source.cloud.google.com/results/invocations/e6dc516f-3a02-494a-b4e4-89014cde6699/targets/maldoca%2Fgcp_ubuntu%2Fcc_test%2Fpresubmit/log